docs/admin/projects.rst

```diff
@@ -115,104 +113,102 @@ Access control
 Enable reviews
     Enable review workflow, see :ref:`reviews`.
 Enable hooks
-    Whether unauthenticated :ref:`hooks` will be enabled for this repository.
+    Whether unauthenticated :ref:`hooks` are to be used for this repository.
 Source language
     Language used for source strings in all components. Change this if you are
     translating from something else than English.
 
 .. note::
 
-    Most of the fields can be edited by project owners or managers in the
+    Most of the fields can be edited by project owners or managers, in the
     Weblate interface.
 
 Adjusting interaction
 +++++++++++++++++++++
 
 There are also additional features which you can control, like automatic
 pushing of changes (see also :ref:`push-changes`) or
-maintaining of Translation-Team header.
+maintainership of the Translation-Team header.
 
 .. _component:
 
 Component configuration
 -----------------------
 
-Component is real component for translating. You enter VCS repository location
-and file mask for which files to translate and Weblate automatically fetches from the VCS
+A component is a grouping of something for translation. You enter a VCS repository location
+and file mask for which files you want translated, and Weblate automatically fetches from this VCS,
 and finds all matching translatable files.
 
 You can find some examples of typical configurations in the :ref:`formats`.
 
 .. note::
 
-    It is recommended to have translation components of reasonable size - split
-    the translation by anything what makes sense in your case (individual
-    applications or addons, book chapters or websites).
+    It is recommended to keep translation components to a reasonable size - split
+    the translation by anything that makes sense in your case (individual
+    apps or addons, book chapters or websites).
 
-    Weblate easily handles translations with 10000 of strings, but it is harder
-    to split work and coordinate among translators with such a large translation.
-    Also when one translator is working on a component, this translation is locked
```

Comment:
Components allow concurrent users now, right?

Reply:
Yes, thanks for spotting.